apps/web/components/dashboard/search/SearchInput.tsx

```diff
@@ -9,11 +9,15 @@ const SearchInput = React.forwardRef<
   React.HTMLAttributes<HTMLInputElement> & { loading?: boolean }
 >(({ className, ...props }, ref) => {
   const { debounceSearch, searchQuery } = useDoBookmarkSearch();
+  const isSearchEnabled = process.env.MEILISEARCH_ENABLED;
```

Comment:
Accessing `process.env` from a client component won't work. If you want it there, you need to add this setting to the client config in `shared/config.ts`, and access it via `useClientConfig()`.